todo.js

```
@@ -1 +1,76 @@
-// This file will be provided as a pull request at the time of your interview.
+const yargs = require('yargs');
+const fs = require('fs');
+
+let toDoList = [];
+
+const loadList = () => {
+    try {
+        const list = require('./to-do-list.json');
+            toDoList = list.sort();
+    } catch (err) {
+        toDoList = [];
```

This suppresses the error. please atleast log it.